lib/src/shared/api.dart

```diff
@@ -0,0 +1,6 @@
+export '../auth/auth.dart'
```

Comment:
Final comment: this needs a license header (you can copy it from any other file and just update the year).